src/components/TaskDetail.jsx

```diff
@@ -104,192 +106,276 @@ export function TaskDetail({ task, isOpen, onClose, onUpdate, onDelete, onAddLab
     onUpdate(task.id, { dueDate: date ? date.toISOString() : null });
   };
 
+  const handleAddComment = () => {
+    if (!newComment.trim() || !user) return;
+    
+    const comment = {
+      id: crypto.randomUUID(),
+      text: newComment.trim(),
+      createdAt: new Date().toISOString(),
+      createdBy: user.uid,
+      createdByName: user.email?.split('@')[0] || 'Anonymous'
+    };
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```suggestion
const generateId = () => {
  try {
    return crypto.randomUUID();
  } catch (e) {
    return Date.now().toString(36) + Math.random().toString(36).substr(2);
  }
};

const comment = {
  id: generateId(),
  text: newComment.trim(),
  createdAt: new Date().toISOString(),
  createdBy: user.uid,
  createdByName: user.email?.split('@')[0] || 'Anonymous'
};
```
The handleAddComment function should include error handling for the crypto.randomUUID() call, as it might not be supported in all browsers. Consider using a fallback mechanism.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;